packages/plugin-conversation/src/interceptor.js

```diff
@@ -196,8 +196,8 @@ export default class ConversationInterceptor extends Interceptor {
           return false;
         }
 
-        if (options.resource !== `content` && options.resource !== `activities` && options.resource !== `conversations`) {
-          return false;
+        if (options.resource !== `content` && options.resource !== `activities` && options.resource !== `conversations` && options.resource !== `teams`) {
+          return options;
```

Comment:
The docs say this function returns a boolean - should it return `true` instead of `options`?